src/Store.js

```diff
@@ -31,9 +30,8 @@ class Store {
 
   @observable currentPage = Login
 
-  constructor(fetch) {
-    this.fetch = fetch
-  }
```

Comment:
Previously, I thought we might want to inject a stubbed-out `fetch` function for use in our tests.
This is the approach outlined in https://hackernoon.com/how-to-decouple-state-and-ui-a-k-a-you-dont-need-componentwillmount-cc90b787aa37

...buuuut, then I discovered [fetch-mock](http://www.wheresrhys.co.uk/fetch-mock/).
It's got a much nicer API than we could create on our own.